package.js

```diff
@@ -1,7 +1,7 @@
 Package.describe({
   name: 'nimble:restivus',
   summary: 'Create authenticated REST APIs in Meteor 0.9+. Setup CRUD endpoints for Collections.',
-  version: '0.6.6',
+  version: '0.6.7',
```

Comment:
Ok, I made this update, and the changelog update.
I am learning git, so, I guess I will create the pull request. Hopefully it is not a duplicate.